x-pack/plugins/observability/public/pages/rules/rules.tsx

```diff
@@ -107,15 +126,21 @@ function RulesPage() {
         <EuiFlexItem>
           <RuleList
             filteredRuleTypes={filteredRuleTypes}
-            showActionFilter={false}
+            lastRunOutcomeFilter={stateLastResponse}
+            refresh={stateRefresh}
             ruleDetailsRoute="alerts/rules/:ruleId"
-            statusFilter={status}
-            onStatusFilterChange={setStatus}
-            lastResponseFilter={lastResponse}
-            onLastResponseFilterChange={setLastResponse}
-            refresh={refresh}
-            rulesListKey={RULES_LIST_COLUMNS_KEY}
-            visibleColumns={RULES_LIST_COLUMNS}
+            rulesListKey="observability_rulesListColumns"
+            showActionFilter={false}
+            statusFilter={stateStatus}
+            visibleColumns={[
+              'ruleName',
+              'ruleExecutionStatusLastDate',
+              'ruleSnoozeNotify',
+              'ruleExecutionStatus',
+              'ruleExecutionState',
+            ]}
+            onLastRunOutcomeFilterChange={handleLastRunOutcomeFilterChange}
```

Comment:
This was broken in the old version. Now changing the last response filter actually changes the URL.